src/components/sequencing/form/ArgTitle.svelte

```diff
@@ -41,15 +43,25 @@
     if (compactTypeInfo) {
       compactTypeInfo = ` [${compactTypeInfo}]`;
     }
-    const base = `${argDef.name}${compactTypeInfo}`;
+    let base = `${argDef.name}${compactTypeInfo}`;
+    if ('range' in argDef && argDef.range) {
+      if (isArray(argDef.range)) {
+        base += ` [${argDef.range.join(', ')}]`;
+      } else {
+        base += ` [${argDef.range.min} – ${argDef.range.max}]`;
+      }
+    }
+
     if ('units' in argDef) {
-      return `${base} - (${argDef.units})`;
+      return `${base} – (${argDef.units})`;
```

Comment:
was this intentionally updated to be the longer dash?

Reply:
Yep